pkg/services/ngalert/notifier/available_channels.go

```diff
@@ -340,5 +340,36 @@ func GetAvailableNotifiers() []*alerting.NotifierPlugin {
 				},
 			},
 		},
+		{
+			Type:        "discord",
+			Name:        "Discord",
+			Heading:     "Discord settings",
+			Description: "Sends notifications to Discord",
+			Options: []alerting.NotifierOption{
+				{
+					Label:        "Message Content",
+					Description:  "Mention a group using @ or a user using <@ID> when notifying in a channel",
+					Element:      alerting.ElementTypeInput,
+					InputType:    alerting.InputTypeText,
+					Placeholder:  `{{ template "default.message" . }}`,
+					PropertyName: "message",
```

Comment:
Suggesting this change for backward compatibility with old config (hence easier migration)
```suggestion
					PropertyName: "content",
```